### PR TITLE
OLS-974: RHOAI VLLM as provider for OLS

### PR DESCRIPTION
**Pull request by tisnik**
## Description

RHOAI VLLM as provider for OLS

## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [x] Unit tests

## Related Tickets & Documents

- Related Issue #[OLS-974](https://issues.redhat.com//browse/OLS-974)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [x] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: ols/src/llms/providers/rhoai_vllm.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            "model": self.model,
+            "model_kwargs": {
+                "top_p": 0.95,
+                "frequency_penalty": 1.03,
+            },
+            "organization": None,
+            "cache": None,
+            "streaming": True,
+            "temperature": 0.01,
+            "max_tokens": 512,
```
Comment (raptorsun) [inline ols/src/llms/providers/rhoai_vllm.py:48]: is 512 tokens too short for generating deployment YAML files?

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

File: ols/src/llms/providers/rhoai_vllm.py (hunk 1 above)
Comment (tisnik) [reply ols/src/llms/providers/rhoai_vllm.py:48]: in this case 512 tokens is approx 280-300 characters (only). So might not be enough

**Pull request merged**